bin/phpunit.sh

```diff
@@ -4,4 +4,4 @@ if [[ $RUN_PHPCS == 1 || $SHOULD_DEPLOY == 1 ]]; then
 	exit
 fi
 
-if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit.xml.dist; else ./vendor/bin/phpunit; fi;
+if [ -f "phpunit.phar" ]; then php phpunit.phar -c phpunit-9.xml.dist; else ./vendor/bin/phpunit; fi;
```

Comment:
Adding the specific configuration file to avoid breaking older PHP Unit versions.